sqlmesh/core/loader.py

```diff
@@ -321,7 +321,7 @@ def _load_sql_models(
     ) -> UniqueKeyDict[str, Model]:
         """Loads the sql models into a Dict"""
         models: UniqueKeyDict[str, Model] = UniqueKeyDict("models")
-        for context_path, config in self._context.configs.items():
+        for context_path, config in self._context._loaders[c.NATIVE][1].items():
```

Comment:
I'm a little concerned it will be annoying to remember what each item in the `_loaders` pairs is, i.e. that `[1]` here refers to the configs dict. Just something to think about in case there's a cleaner way to implement it.

Reply:
was wondering the same, do you think a dictionary instead of tuple would be better with loader and configs keys to distinguish it better?
so doing then something like: `self._context._loaders[c.NATIVE].configs`

Reply:
Yes, this makes sense to me 👍